app/src/main/java/com/yagni/model/ReadAllOrders.java

```diff
@@ -16,7 +17,6 @@ public ReadAllOrders(JTable ordersTableIn){
     public void readAllOrders(DbConnection linkDB) {
         
         try {
-            
             Statement state = linkDB.getConnection().createStatement(); // creates a statment object
```

Comment:
move your comment that trails the code at line 20 up, the comment should be on line 20 and the code on line 21. Also, statement is misspelled in the comment.
like this:
line 20 --> // creates a statement object
line 21 -->Statement state = linkDB.getConnection().createStatement();